docs/v2-system-addon/data_events.md

```diff
@@ -1,6 +1,14 @@
 # Metrics we collect
 
-This is an overview of the different kinds of data we collect in Activity Stream experiment. See [data_dictionary.md](data_dictionary.md) for more details for each field.
+By default, the about:newtab and about:home pages in Firefox (the pages you see when you open a new tab and when you start the browser), will send data back to Mozilla servers about usage of these pages.  The intent is to collect data in order to improve the user's experience while using Activity Stream.  Data about your specific browsing behaior or the sites you visit is **never transmitted to any Mozilla server**.  At any time, it is easy to **turn off** this data collection by [opting out of Firefox telemetry](https://support.mozilla.org/kb/share-telemetry-data-mozilla-help-improve-firefox).
+
```

Comment:
Yes, agree with everything you've just said.  However, the SUMO URL above shows people a picture of how to toggle  "Share additional data/Telemetry".  To toggle the data we're sharing, one needs to switch the "FHR" toggle, which appears to be documented at [https://support.mozilla.org/en-US/kb/firefox-health-report-understand-your-browser-perf](https://support.mozilla.org/en-US/kb/firefox-health-report-understand-your-browser-perf).